download-json-simple.sh

```diff
@@ -0,0 +1,106 @@
+#!/usr/bin/env bash
```

Comment:
Agreed. I went with the bash script for now since I didn't want to duplicate whatever work she was already doing for Windows support. Once she's got something that works, we can apply it to this.